### ISSUE TITLE
`quicer:open_lib()` crashes with `noproc` if called early in node boot

### ISSUE THREAD

**Issue by pjk25** [emqx/quic#80]
Taking a look at using quic in an "Alternative Carrier for the Erlang Distribution", and it appears that `file:read_file_info/1` cannot be called that early in node startup. The crash I see originates from https://github.com/emqx/quic/blob/0742e6dde38a09cd7fb9461d6c0db051938c7939/src/quicer_nif.erl#L217 and results in the following error:

```
The on_load function for module quicer_nif returned:
{noproc,{gen_server,call,[file_server_2,{read_file_info,[...]},infinity]}}
```

**Comment by qzhuyan** [comment]
@pjk25 thanks, run Erlang Distribution over QUIC is also in my TODO list. Could you share your boot script? running on linux right?

**Comment by pjk25** [comment]
You'll have clarify what you mean by "boot script" for me. Currently I'm just starting a couple of erlang shells interactively and testing with those, so `erl -sname lead -proto_dist inet_quic` with an appropriate `ERL_LIBS`. So I guess that implies the default boot script?

So far testing this on macos, and I have since realized that `quicer:accept_stream/3` times out when I follow the example in the README. `make test` does pass, though.

**Comment by qzhuyan** [comment]
> So I guess that implies the default boot script?
Yes. I just want to ensure we are looking at the same boot script.


>quicer:accept_stream/3 times out
Are you on master branch? 
note that client side must send something on the started stream otherwise the stack will not init the stream to the remote server side.
